client/src/BibleEngineClient.ts

```diff
@@ -1,4 +1,5 @@
 import { ConnectionOptions } from 'typeorm';
+import 'cross-fetch/polyfill';
```

Comment:
i wonder if it is good practice to include a polyfill on this level. i'd suggest doing that in your entry-file (possibly via one file where you collect all polyfill imports)

Reply:
polyfills are dependent on where you run the code. i think it's not uncommon for a library to put this in the README rather than actually importing it. a lot of projects already import a polyfill for `fetch` so this would potentially cause problems or at least be redundant.

Reply:
Agree. done